ui/component/app/view.jsx

```diff
@@ -261,8 +261,12 @@ function App(props: Props) {
   }, [appRef, fetchAccessToken, fetchChannelListMine, fetchCollectionListMine]);
 
   useEffect(() => {
-    // $FlowFixMe
-    document.documentElement.setAttribute('theme', theme);
+    if (!pathname.includes('/$/overlay/')) {
+      /* This is needed otherwise the Overlay page will not work correctly. */
+      /* If pathname does not include Overlay, set the following attributes. */
+      document.documentElement.setAttribute('theme', theme);
+      document.documentElement.setAttribute('dir', 'ltr');
+    }
```

Comment:
See note about moving the page outside of the page.  Don't pollute the app.

Reply:
See [above](https://github.com/OdyseeTeam/odysee-frontend/pull/348#discussion_r780923166).